cli/tests/pcluster/config/test_validators.py

```diff
@@ -516,25 +576,48 @@ def test_kms_key_validator(mocker, boto3_stubber):
 
 
 @pytest.mark.parametrize(
-    "section_dict, expected_message",
+    "section_dict, bucket, expected_message",
     [
-        ({"imported_file_chunk_size": 1024, "import_path": "test", "storage_capacity": 1200}, None),
+        (
+            {"imported_file_chunk_size": 1024, "import_path": "s3://test", "storage_capacity": 1200},
+            {"Bucket": "test"},
+            None,
+        ),
         (
             {"imported_file_chunk_size": 1024, "storage_capacity": 1200},
+            None,
             "When specifying 'imported_file_chunk_size', the 'import_path' option must be specified",
         ),
-        ({"export_path": "test", "import_path": "test", "storage_capacity": 1200}, None),
+        ({"export_path": "s3://test", "import_path": "s3://test", "storage_capacity": 1200}, {"Bucket": "test"}, None),
         (
-            {"export_path": "test", "storage_capacity": 1200},
+            {"export_path": "s3://test", "storage_capacity": 1200},
+            {"Bucket": "test"},
             "When specifying 'export_path', the 'import_path' option must be specified",
         ),
-        ({"shared_dir": "NONE", "storage_capacity": 1200}, "NONE cannot be used as a shared directory"),
-        ({"shared_dir": "/NONE", "storage_capacity": 1200}, "/NONE cannot be used as a shared directory"),
-        ({"shared_dir": "/fsx"}, "the 'storage_capacity' option must be specified"),
-        ({"shared_dir": "/fsx", "storage_capacity": 1200}, None),
+        ({"shared_dir": "NONE", "storage_capacity": 1200}, None, "NONE cannot be used as a shared directory"),
+        ({"shared_dir": "/NONE", "storage_capacity": 1200}, None, "/NONE cannot be used as a shared directory"),
+        ({"shared_dir": "/fsx"}, None, "the 'storage_capacity' option must be specified"),
+        ({"shared_dir": "/fsx", "storage_capacity": 1200}, None, None),
     ],
 )
-def test_fsx_validator(mocker, section_dict, expected_message):
+def test_fsx_validator(mocker, boto3_stubber, section_dict, bucket, expected_message):
+    head_bucket_response = {
+        "ResponseMetadata": {
+            "AcceptRanges": "bytes",
+            "ContentType": "text/html",
+            "LastModified": "Thu, 16 Apr 2015 18:19:14 GMT",
+            "ContentLength": 77,
+            "VersionId": "null",
+            "ETag": '"30a6ec7e1a9ad79c203d05a589c8b400"',
+            "Metadata": {},
+        }
+    }
+    mocked_requests = [MockedBoto3Request(method="head_bucket", response=head_bucket_response, expected_params=bucket)]
```

Comment:
We can create an utility method to mock s3_head_bucket call and use it everywhere. The method can have the bucket name as parameter and return the mocked_requests to use in the boto3_stubber

Reply:
good idea

Reply:
done